src/main.c

```diff
@@ -1540,7 +1541,8 @@ static gchar* r_status_formatter_readable(RaucStatusPrint *status)
 	g_string_append(text, "=== System Info ===\n");
 	g_string_append_printf(text, "Compatible:  %s\n", status->compatible);
 	g_string_append_printf(text, "Variant:     %s\n", status->variant);
-	g_string_append_printf(text, "Booted from: %s (%s)\n\n", bootedfrom ? bootedfrom->name : NULL, status->bootslot);
+	g_string_append_printf(text, "Booted from: %s (%s)\n", bootedfrom ? bootedfrom->name : NULL, status->bootslot);
```

Comment:
In the commit message:

```
    src/main: add prevent-late-fallback to userspace tools
```

Since RAUC only runs in 'userspace', I'd leave that term out. The commit also doesn't add 'prevent-late-fallback', it just adds output for counter locking. Essentially, you `add counter locking information to rauc status output`.

```
    This adds the option to print the current state of prevent-late-fallback
    and also to enable and disable sets the lock-counter via userspace.
```

The sentence looks inconsistent. Also, the corresponding commit does not add any functionallity to *enable/disable/set* something.

```
    The possibility to get/set it via D-Bus is not yet
    implemented and will be added separately.
```